src/components/fx/hover.js

```diff
@@ -25,6 +25,8 @@ var Registry = require('../../registry');
 var helpers = require('./helpers');
 var constants = require('./constants');
 
+var MAJOR_VERSION = 1; // i.e. Plotly version
```

Comment:
Nop. Please don't include version-specific logic. Write code for `v1.x`, we'll patch this up in a `v2` branch when we get there.